org.eclipse.xtext.common.types.tests/tests/org/eclipse/xtext/common/types/access/impl/AbstractTypeProviderTest.java

```diff
@@ -3530,7 +3530,8 @@ public void testFindTypeByName_AbstractMultimap() {
 		String typeName = "com.google.common.collect.AbstractMultimap";
 		JvmGenericType type = (JvmGenericType) getTypeProvider().findTypeByName(typeName);
 		assertNotNull(type);
-		diagnose(type, "java:/Objects/javax.annotation.Nullable#javax.annotation.Nullable");
+		diagnose(type, "java:/Objects/javax.annotation.Nullable#javax.annotation.Nullable",
```

Comment:
@ArneDeutsch this seem to be unrelated and caused by different classpaths.
thus i assume this will fail with gradle

Reply:
Maybe we should exclude errorprone in the pom?

Reply:
am not sure where this is coming from on arnes machine

Reply:
will test this on my windows machine in the evening

Reply:
@szarnekow i dont understand the test. there is a guard
`if (diagnostic.getSeverity() != Diagnostic.OK) {` in   `org.eclipse.xtext.common.types.access.impl.AbstractTypeProviderTest.diagnose(EObject, String...)  `
so that if there is no diagnostic at all the test will work regardless if there are any expected diagnostics or not.
is this behaviour intended? if no why does it behave differently on windows (or does it on your branch only @ArneDeutsch )

Reply:
In different classpath configurations, we do have different missing transitive types. `diagnose(EObject, String...)` is a poor attempt to whitelist missing types.

Reply:
It would be great if we could get rid of this.